Cartfile.resolved

```diff
@@ -1,3 +1,3 @@
-github "Quick/Nimble" "v6.1.0"
+github "Quick/Nimble" "v7.0.0"
 github "Quick/Quick" "v1.1.0"
-github "ReactiveX/RxSwift" "3.4"
+github "ReactiveX/RxSwift" "3.5.0"
```

Comment:
Sorry for late reply.
`v3.4` of the library is sufficient, but I think that development should always be done with the latest version.
This change only changes the default version and does not force the user to `v3.5`.